common/pinot/pinotClient.go

```diff
@@ -153,7 +151,7 @@ func (c *PinotClient) getInternalGetClosedWorkflowExecutionResponse(resp *pinot.
 	schema := resp.ResultTable.DataSchema // get the schema to map results
 	columnNames := schema.ColumnNames
 	actualHits := resp.ResultTable.Rows
-	response.Execution = ConvertSearchResultToVisibilityRecord(actualHits[0], columnNames, c.logger, isSystemKey)
+	response.Execution = ConvertSearchResultToVisibilityRecord(actualHits[0], columnNames, c.logger)
```

Comment:
The isSystemKey() method below can also be removed

Reply:
Do we no longer need any kind of system key check or it is required when build the query?

Reply:
Yes we don't need it anymore since it is already be checked in frontend. We can be 100% sure that those are valid.